soda-consumer-scala/src/main/scala/com/socrata/soda2/consumer/impl/RowDecoder.scala

```diff
@@ -77,6 +76,11 @@ object RowDecoder {
     "geospatial" -> SodaGeospatial,
     "location" -> SodaLocation,
     "point" -> SodaLocation,
+    "multi_point" -> SodaGeospatial,
+    "line_string" -> SodaGeospatial,
+    "multi_line_string" -> SodaGeospatial,
+    "polygon" -> SodaGeospatial,
+    "multi_polygon" -> SodaGeospatial,
```

Comment:
This isn't a super useful decoding of these more-specific types.  It basically just leaves "extract stuff from the geojson format" up to the user.

Reply:
Problem was not having the type here and therefore getting and `UnknownTypeException`. I think covering them with `SodaGeospatial` type is enough for the scope of this ticket. We can cover the rest in the future when there is a ticket for that.